settings.gradle

```diff
@@ -25,6 +25,7 @@ include 'vividus-plugin-db'
 include 'vividus-plugin-excel'
 include 'vividus-plugin-electron'
 include 'vividus-plugin-html'
+include 'vividus-plugin-json'
```

Comment:
also please add the plugin here: https://github.com/vividus-framework/vividus/blob/master/vividus-docker-bundler/build.gradle